cmd/createtree/main_test.go

```diff
@@ -218,6 +232,9 @@ func (s *fakeAdminServer) CreateTree(ctx context.Context, req *trillian.CreateTr
 		return nil, s.err
 	}
 	resp := *req.Tree
+	if req.KeySpec != nil && s.generatedKey != nil {
```

Comment:
True, but in this case I think it's more about mimicking AdminServer and less about the test checking itself. This is a suggestion that doesn't change much in practice right now, so feel free to ignore it.